docker-compose.yml

```diff
@@ -5,32 +5,28 @@ volumes:
 services:
   tesla_ble_mqtt:
     container_name: tesla_ble_mqtt
-    image: "tesla-local-control/tesla_ble_mqtt:latest"
+    image: "iainbullock/tesla_ble_mqtt:latest"
+    #image: "tesla_ble_mqtt:latest"
 
-
-    # Configuration Settings have been moved to the "env" file
+    # Configuration Settings have been moved to the "stack.env" file
     env_file:
-      - env
+      - stack.env
 
+    # Needed to open the Bluetooth dongle to send commands to the car
     network_mode: host
-
-    # TODO:  Needed to open the Bluetooth dongle to send commands to the car
     privileged: true
-    # TODO: Needed to open the Bluetooth device
-    tty: true
 
-    # TODO : Why?
+    # Allow opening of a shell in container
+    tty: true
     stdin_open: true
 
-    # TODO : still valid?
-    working_dir: /share/tesla_ble_mqtt
-
     volumes:
       - tesla_ble_mqtt:/share/tesla_ble_mqtt
       - /run/dbus:/run/dbus
 
-    # TODO: most likely not needed to overwrite the image's default
-    entrypoint: "/app/run.sh"
+    # Uncomment if you need a container shell when the container won't run 
+    # entrypoint: "/bin/ash"
 
-    restart: no
+    # Uncommenting restart: unless-stopped is recommended when the container is fully configured and working correctly    
```

Comment:
then should you uncomment below by default?

Reply:
I prefer it to remain as "no" by default as it's more obvious for NOOBs that it's stopped